WordPress/Classes/ViewRelated/Views/WPBlogSelectorButton.m

```diff
@@ -20,7 +20,6 @@ + (instancetype)buttonWithFrame:(CGRect)frame buttonStyle:(WPBlogSelectorButtonS
 
     button.titleLabel.adjustsFontSizeToFitWidth = NO;
     [button setImage:[UIImage imageNamed:@"icon-nav-chevron"] forState:UIControlStateNormal];
-    [button setAccessibilityHint:NSLocalizedString(@"Tap to select which blog to post to", @"This is the blog picker in the editor")];
```

Comment:
> @ceyhun I am thinking to do the cleanup/removal in a separate PR to keep the implementation clean in this one & I am on it 🚀
In this PR, I have removed just the source, which is blogPickerBarButtonItem

Sound good to me 👍 Should we maybe leave this change to that PR as well? Or is this related to this one?

Reply:
👍 

> Should we maybe leave this change to that PR as well? Or is this related to this one?

I think it is 😄, this line was not really needed, it was getting ignored by the OS as `AccessibilityLabel` of **barButtonItem** was sufficient for the user.

**Refs**
Stackoverflow: [VoiceOver Accessibility accessibilityHint not announced](https://stackoverflow.com/questions/48299040/voiceover-accessibility-accessibilityhint-not-announced)
Apple Documentation [Accessibilityhint](https://developer.apple.com/documentation/objectivec/nsobject/1615093-accessibilityhint)

Reply:
> I'm still not sure why this is related to this PR. I think the WPBlogSelectorButton will become unused when this PR is merged. So why are we modifying it?

Yeah, You are right 👍, will revert

> When I try it on current develop branch it is not ignored. I tested on iOS 14.4 with an iPhone 11 Pro device. In which case was it ignored for you?

My bad 😃 that I tested on simulator
Tried on Device it's working perfectly fine

Reply:
> Yeah, You are right 👍, will revert

Fixed at a9291203265780ea296548499797310ccd9d554d